crates/ruma-api/src/lib.rs

```diff
@@ -135,9 +135,10 @@ use ruma_identifiers::UserId;
 ///             description: "Does something.",
 ///             method: POST,
 ///             name: "some_endpoint",
-///             path: "/_matrix/some/endpoint/:baz",
+///             stable: "/_matrix/some/endpoint/:baz",
```

Comment:
The keywords for the new fields still don't end in `_path`?

Reply:
the keywords dont, i thought you were completely fine with this in https://github.com/ruma/ruma/pull/859?

Reply:
after some discussion in chat, the keyword will be converted to use `_path` as well